### PR TITLE
Adding caisy as cms to the docs

### PR DESCRIPTION
**Pull request by MoritzGruber**
Hi there, thank you for maintaining Astro and its documentation! I love the framework! As a founding member of caisy, I would like to submit some additions to your docs regarding our integration with Astro.

#### What kind of changes does this PR include?
Changes to the docs content. 

#### Description
Adding caisy as cms/service to the docs. 
Our homepage: https://caisy.io

- What does this PR change? 
Adding caisy to the overview of cms's and adding a new subpage for caisy
- Did you change something visual? 
Before:
before
After:
after

Thanks for reviewing and best regards, 
Moritz

### CODE REVIEW

**Comment by TheOtterlord** [comment]
Thanks for this CMS guide @MoritzGruber!

I've restructured the guide to better match our other CMS guides, and the layout we've decided on, as well as mark it as a stub, to let people know we're happy for someone to expand it into an in-depth guide!

**Comment by sarah11918** [comment]
@MoritzGruber Thank you so much for this contribution! We love having external contributions, and want as many people to enjoy working with Astro as possible!  :raised_hands: 

Great job adjusting this to our format, @TheOtterlord ! I am happy to approve the English version immediately!

@MoritzGruber - The German page needs to be reworked to match the English before we can merge this. OR, you are welcome to remove the German for now, and we'll merge the English immediately. Then, you can PR a German translation. The choice is yours! Just let us know!

**Comment by MoritzGruber** [comment]
@sarah11918, I've removed the German version for now. I think it's simpler to move this forward without it first. I'll create another PR for the translation. Thanks for the hint.

File: src/content/docs/en/guides/cms/caisy.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+service: Caisy
+i18nReady: true
+stub: true
+---
+
+[Caisy](https://caisy.io/) is a headless CMS that exposes a GraphQL API to access content.
+
+## Using Caisy CMS with Astro
+
+A data fetching example using `graphql-request` and Caisy's rich text renderer for Astro.
```
Comment (sarah11918) [inline src/content/docs/en/guides/cms/caisy.mdx:14]:
```suggestion
Use `graphql-request` and Caisy's rich text renderer for Astro to fetch your CMS data and display your content on an Astro page:
```

**Comment by sarah11918** [comment]
Great, @MoritzGruber, then we're happy to add you (and Caisy) to Team Docs!  :partying_face:

**Pull request merged**